libs/scraper-agent/src/processors/accountSummary.test.ts

```
@@ -50,26 +51,57 @@ describe ("Correctly finds the balance element", () => {
 
 
 describe("Correctly finds the navigation element", () => {
-  const testData = getTestData("AccountsSummary", "navigate", "latest/TD");
-  it.each(testData)("For: %s", async (test) => {
-    await using agent = await test.agent();
+  const testData = getTestData("AccountsSummary", "navigate", "latest");
+  let candidates: TestElmData[]|undefined;
+  EventBus.get().onElement((_elm, _params, all) => {
+    candidates = all;
+  })
+
```

ESLint

[error] Missing semicolon.

(semi)